week1/3-website/css/style.css

```diff
@@ -6,3 +6,176 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+
+body {
+    margin: 20px;
+    line-height: 1.5; /* to create gaps between lines for easy readying*/
+}
+
+ header, footer {
+     color: rgb(17, 80, 17);
+ }
+ 
+ /*BUG: I tried header:not(ul) to select everything in header but not ul but it didn't work*/
+ h1, h2, footer {
+     text-align: center;
+ }
+
+ /*article subheadings*/
+ h3 {
+    color: rgb(30, 139, 30);
+    text-shadow: 4px 4px 6px rgb(80, 60, 17);
+ }
+
+ /*title cooked*/
+ .cooked {
+     text-decoration: line-through;
+     background: rgb(0, 0, 0);
+     color: rgb(255, 255, 255);
+     transform: rotate(-30deg); /*BUG not working - I want to rotate the text in the title*/
+ }
+
+/*BUG: linear-gradient not working*/
+ footer {
+    background: linear-gradient (rgb(0, 255, 0), rgb(255, 255, 255));
+ }
+
+article:first-child {
+    background: rgba(0, 255, 0, 0.171);
+}
+
+article {
+    padding: 1px 30px 1px 30px;
+}
+
+p:last-of-type {
+    color: rgb(80, 60, 17);
+}
+
+p:first-of-type:first-letter {
+    font-weight: bold;
+    font-size: 1.3em;
+    font-style: oblique;
+    color: rgb(80, 60, 17);
+}
+
+/*for accessibility*/
+a:focus {
+    background: rgba(255, 238, 0, 0.5);
+}
+
+a:visited {
+    color: rgb(126, 30, 139);
+}
+
+a:hover {
+    font-weight: bold;
+}
+
+/*first 3 <a> tags*/
+li a:nth-child(-n+3) {
+    font-size: 20px;
+    color: rgb(30, 139, 30);
+    text-decoration: none;
+    background: rgb(80, 60, 17, .1);
+    border-radius: 50px;
+    padding: 2px;
+    font-variant: small-caps;
+}
+
+/*Back to top link*/
+br + a {
+    font-size: 15px;
+    color: rgb(30, 139, 30);
+    background: rgb(80, 60, 17, .1);
+    border-radius: 50px;
+    padding: 5px;
+    font-variant: small-caps;
+    text-decoration: none;
+    text-align: center;         /*BUG text-align doesn't work so I use display: table; and margin: auto;*/
```

Comment:
Anchor elements are inline elements, and you can't set a `text-align` property directly on them, what you usually want is to set that property on their parent. But In your case that would make extra text centered as it is inside `<p>` element which contains more text. You can wrap that anchor element within a container `div` and give the container `text-align: center` property.
But your method with `display: table` and `margin: auto` works perfectly fine as well. Also you can choose a `display: block` and then give the width for a button (you still would need a `margin: auto`).